### PR TITLE
[UX] Sending at, Activate at and Available from (statuses)

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This PR aligns the schedule options with the status label used by a bundle. Example: Since you make an Asset available for use, it makes sense to schedule "Available from" and "Unavailable from" instead of the generic "Activate at/Deactivate at".

All changes:
- Fixes the broken schedule logic for Emails. When you create a Template Text Message (sms), the "Publish/Activate at" options are hidden. They should also be hidden for Template Emails.
- Aligns Segment Emails and SMS to use "Start sending at" since this way better expresses the functionality 
- Uses "Available from" instead of "Activate at" for resources where the publish label is "Available for use" to improve consistency and feedback (users create a better relationship between the fields)

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Plugins > Twilio > Fill anything and set Active > Save
3. Open Components > Emails > New > Template > See any "Start sending at" options > Save
4. Repeat the step 3 for Template Text Messages (SMS)
5. Open Components > Emails > New > Segment > See "Start sending at" options  > Save
6. Repeat the step 5 for Segment Text Messages (SMS)
7. Open Components > Emails > Open the created Template Email > Dropdown > Add A/B test > See no "Start sending at" options
8. Open Components > Emails > Open the created Segment Email > Dropdown > Add A/B test > See "Start sending at" options
9. Open Components > Assets > New > See "Available from" options
10. Repeat the step 9 for Marketing Messages, Forms and Landing pages

### CODE REVIEW

File: app/bundles/AssetBundle/Resources/views/Asset/form.html.twig [hunk 1]
```diff
@@ -80,8 +80,8 @@
 			{{ form_row(form.category) }}
 			{{ form_row(form.language) }}
 			{{ form_row(form.isPublished) }}
-			{{ form_row(form.publishUp) }}
-			{{ form_row(form.publishDown) }}
+			{{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
+    		{{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```
Comment (shinde-rahul) [inline app/bundles/AssetBundle/Resources/views/Asset/form.html.twig:84]:
```suggestion
			{{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
    		        {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```

File: app/bundles/PageBundle/Resources/views/Page/form.html.twig [hunk 2]
```diff
@@ -111,8 +111,8 @@
                   {{ form_row(form.isPreferenceCenter) }}
               {% endif %}
 
-              {{ form_row(form.publishUp) }}
-              {{ form_row(form.publishDown) }}
+              {{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
+    		  {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```
Comment (shinde-rahul) [inline app/bundles/PageBundle/Resources/views/Page/form.html.twig]:
```suggestion
              {{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
    	     {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```

**Review by shinde-rahul — CHANGES_REQUESTED** [review]
Needs cs fixes.

**Comment by andersonjeccel** [comment]
@shinde-rahul Thanks for fixing 🙏🏻

**Review by shinde-rahul — APPROVED** [review]
LGTM!!!

**Review by LeonOltmanns — APPROVED** [review]
LGMT from a User-Perspective.

File: app/bundles/AssetBundle/Resources/views/Asset/form.html.twig [hunk 3]
```diff
@@ -80,8 +80,8 @@
 			{{ form_row(form.category) }}
 			{{ form_row(form.language) }}
 			{{ form_row(form.isPublished) }}
-			{{ form_row(form.publishUp) }}
-			{{ form_row(form.publishDown) }}
+			{{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
+    		        {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```
Comment (escopecz) [inline app/bundles/AssetBundle/Resources/views/Asset/form.html.twig]: Please fix the indentation.

File: app/bundles/EmailBundle/Assets/js/email.js [hunk 4]
```diff
@@ -236,11 +236,13 @@ Mautic.selectEmailType = function(emailType) {
         mQuery('#segmentTranslationParent').removeClass('hide');
         mQuery('#templateTranslationParent').addClass('hide');
         mQuery('.page-header h3').text(mauticLang.newListEmail);
+        mQuery('#scheduleOptions').removeClass('hide');
```
Comment (escopecz) [inline app/bundles/EmailBundle/Assets/js/email.js:239]: Why to hide it for both list and template types? It's the all types there are. From the programming perspective, it shouldn't be in the if nor else blocks but bellow it as you want to apply it to both. But from the product perspective, why do we have a feature that we want to hide for all options?

File: app/bundles/PageBundle/Resources/views/Page/form.html.twig [hunk 5]
```diff
@@ -111,8 +111,8 @@
                   {{ form_row(form.isPreferenceCenter) }}
               {% endif %}
 
-              {{ form_row(form.publishUp) }}
-              {{ form_row(form.publishDown) }}
+              {{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
+    	     {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```
Comment (escopecz) [inline app/bundles/PageBundle/Resources/views/Page/form.html.twig]: Another indentation issue.

**Review by escopecz — CHANGES_REQUESTED** [review]

File: app/bundles/EmailBundle/Assets/js/email.js (hunk 4 above)
Comment (escopecz) [reply app/bundles/EmailBundle/Assets/js/email.js:239]: Oops, this is `removeClass`. Makes sense then.

File: app/bundles/AssetBundle/Resources/views/Asset/form.html.twig (hunk 3 above)
Comment (escopecz) [reply app/bundles/AssetBundle/Resources/views/Asset/form.html.twig]:
```suggestion
			{{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
    		{{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```

File: app/bundles/PageBundle/Resources/views/Page/form.html.twig (hunk 5 above)
Comment (escopecz) [reply app/bundles/PageBundle/Resources/views/Page/form.html.twig]:
```suggestion
              {{ form_row(form.publishUp, {'label': 'mautic.core.form.available.available_from'}) }}
    	      {{ form_row(form.publishDown, {'label': 'mautic.core.form.available.unavailable_from'}) }}
```

**Review by escopecz — APPROVED** [review]
I fixed the indentations 👍

**Pull request merged**